e2e_playwright/st_caption_test.py

```diff
@@ -34,6 +34,21 @@ def test_correct_content_in_caption(app: Page):
     )
 
 
+def test_help_tooltip_works(app: Page):
+    """Test that the help tooltip is displayed on hover."""
+    # The stMarkdown div is the outermost container that holds the caption and the help tooltip:
+    caption_with_help = app.get_by_test_id("stMarkdown").nth(3)
+
+    hover_target = caption_with_help.get_by_test_id("stTooltipHoverTarget")
+    expect(hover_target).to_be_visible()
+
```

Comment:
nit: I think it is a good idea to have a non-attached check here as well to ensure that the `hover` is doing something, like:

```python
tooltip_content = app.get_by_test_id("stTooltipContent")
expect(tooltip_content).to_not_be_attached()

hover_target.hover()

expect(tooltip_content).to_be_visible()
```